src/main/java/de/sldk/mc/metrics/Entities.java

```diff
@@ -2,22 +2,50 @@
 
 import io.prometheus.client.Gauge;
 import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
 import org.bukkit.plugin.Plugin;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 public class Entities extends WorldMetric {
 
     private static final Gauge ENTITIES = Gauge.build()
             .name(prefix("entities_total"))
             .help("Entities loaded per world")
-            .labelNames("world")
+            .labelNames("world", "type", "alive")
             .create();
 
+    /**
+     * Override the value returned by {@link EntityType#isAlive()}.
+     */
+    private static final Map<EntityType, Boolean> aliveOverride = new HashMap<EntityType, Boolean>() {{
+        put(EntityType.ARMOR_STAND, false);
+    }};
```

Comment:
I use uppercase names for constants. You could also change it to this to get an immutable map:

```java
private static final Map<EntityType, Boolean> ALIVE_OVERRIDE = Collections.singletonMap(EntityType.ARMOR_STAND, false);
```